studio/schemas/objects/link.ts

```diff
@@ -191,7 +193,7 @@ export const link = defineField({
     prepare(selection) {
       const { title, type } = selection;
       return {
-        title: title,
+        title: title[0].value,
```

Comment:
To not overcomplicate this I've set it to index [0]. That might be an English translation or a Norwegian, but not something we can spend time on now.